inc/cappuccino/UtlruCache.tcc

```diff
@@ -176,16 +176,24 @@ auto UtlruCache<KeyType, ValueType, SyncType>::doInsertUpdate(
     const KeyType& key,
     ValueType&& value,
     std::chrono::steady_clock::time_point now,
-    std::chrono::steady_clock::time_point expire_time) -> void
+    std::chrono::steady_clock::time_point expire_time) -> bool
 {
     auto keyed_position = m_keyed_elements.find(key);
     if (keyed_position != m_keyed_elements.end()) {
-        doUpdate(keyed_position, std::move(value), expire_time);
+        // Check whether expired
+        const size_t element_idx = keyed_position->second;
+        const bool expired = (now >= m_elements[element_idx].m_expire_time);
+
+        Element& element = doUpdate(keyed_position, expire_time);
```

Comment:
if this always unconditionally updates or sets the value, what use is the return value?

Reply:
Oops -- will fix

Reply:
Fixed